src/17alasql.js

```diff
@@ -10,7 +10,7 @@
 alasql.parser = alasqlparser;
 
 /*/* This is not working :-/ */
-alasql.parser.parseError = function (str, hash) {
+alasqlparser.parseError = function (str, hash) {
```

Comment:
yes it was (to reduce the number of properties lookup). you where doing something in lines of:
```js
alasql.parser = alasqlparser;
alasql.parser.parseError = function (str, hash) {
	...
}
```
it would also be okey set the parseError before assigning it to alasql
```js
alasqlparser.parseError = function (str, hash) {...}
alasql.parser = alasqlparser
```

Reply:
Lets not introduce breaking changes just to cut one dot out of the source code

Reply:
```diff
- alasqlparser.parseError = function (str, hash) {
+ alasql.parser.parseError = function (str, hash) {
```

Reply:
reverted